Print/fcl/printLong.fcl

```diff
@@ -72,12 +72,17 @@ physics :{
 	#inputTags : [ "CaloClusterMaker" ] # select instances
       }
 
+      crvStepPrinter : {
+        verbose: 1      # 0.1
+        eCut : -10.0          # cut on hits
```

Comment:
Is this negative sign correct (no energy cut)?

Reply:
Yes, negative means effectively no cut.  It is here as a reminder that the cut is possible.

Reply:
Ok